src/ccall.cpp

```diff
@@ -1590,22 +1599,24 @@ static jl_cgval_t emit_ccall(jl_value_t **args, size_t nargs, jl_codectx_t *ctx)
             largty = julia_struct_to_llvm(tti, unionall, &isboxed);
         }
         if (isboxed) {
-            ary = boxed(emit_expr(argi, ctx), ctx);
+            ary = boxed(ctx, emit_expr(ctx, argi));
         }
         else {
             assert(!addressOf);
-            ary = emit_unbox(largty, emit_expr(argi, ctx), tti);
+            ary = emit_unbox(ctx, largty, emit_expr(ctx, argi), tti);
         }
         JL_GC_POP();
         if (!retboxed) {
             return mark_or_box_ccall_result(
-                      emit_bitcast(emit_pointer_from_objref(
-                                          emit_bitcast(ary, T_prjlvalue)), lrt),
-                   retboxed, rt, unionall, static_rt, ctx);
+                    ctx,
```

Comment:
visually distinct from what? it obviously isn't control flow by virtue of calling something else and not introducing a new block.